include/meta/topics/default_term_scorer.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file topic_model.h
+ * @author Matt Kelly
+ *
+ * All files in META are dual-licensed under the MIT and NCSA licenses. For more
+ * details, consult the file LICENSE.mit and LICENSE.ncsa in the root of the
+ * project.
+ */
+
+#ifndef META_TOPICS_DEFAULT_TERM_SCORER_H_
+#define META_TOPICS_DEFAULT_TERM_SCORER_H_
+
+#include "meta/index/forward_index.h"
+#include "meta/meta.h"
+#include "meta/topics/topic_model.h"
+
+namespace meta
+{
+namespace topics
+{
+
+/* Scores terms in some way Chase can probably articulate and or source */
```

Comment:
It should be equivalent to the Blei and Lafferty scoring function, just using log properties to (hopefully?) be faster.

If we can get this working, I think it should be called `bl_term_scorer` since that's really what it is.